### PR TITLE
add thumbnail images to archive command embeds

### PR DESCRIPTION
**Pull request by Godlander**
## Describe your changes

add thumbnail images to archive command embeds

## Testing/validation

image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: apps/somnia/src/assets/assets.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export const rarityColors = [0x818486, 0x5a977a, 0x5987ad, 0x9470bb, 0xc87c24]
+export const elementAssets = {
+  none: {
+    img: 'https://api.ambr.top/assets/UI/UI_Icon_Item_Temp.png',
```
Comment (nguyentvan7) [inline apps/somnia/src/assets/assets.ts:4]: should also add these to AssetData.ts

File: apps/somnia/src/assets/assets.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export const rarityColors = [0x818486, 0x5a977a, 0x5987ad, 0x9470bb, 0xc87c24]
```
Comment (nguyentvan7) [inline apps/somnia/src/assets/assets.ts:1]: we have these in libs/gi/theme/src/Theme.tsx

If we want to be consistent, we can have a libs/gi/color library or something so frontend and bot can share the same rarity colors

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]
various style comments

File: apps/somnia/src/assets/assets.ts (hunk 2 above)
Comment (Godlander) [reply apps/somnia/src/assets/assets.ts:1]: i think that would be better, bot cant use tsx

**Pull request closed without merging**